golang/example/example_lambda.go

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2020 - present Kurtosis Technologies LLC.
+ * All Rights Reserved.
+ */
+
+package example
+
+type ExampleLambda struct {
+	isKurtosisCoreDevMode bool
```

Comment:
Do we need this? This exists in the testsuite purely because a) free users are limited to 3 tests b) the example testsuite also contains the Kurtosis-internal tests, which are much > 3 c) we bootstrap from the example tests, so we if we didn't have the isKurtosisCoreDevMode flag then free users would bootstrap a testsuite that they can't use